.github/workflows/test.yml

```diff
@@ -157,7 +157,8 @@ jobs:
           file_system: ${{ matrix.file_system }}
           tpm_enabled: true
           suite: "eve-upgrade.tests.txt"
-          eve_image: ${{ inputs.eve_image }}
+          eve_image: lfedge/eve:11.0.5-lts
+          env_vars: EVE_VERSION=${{ inputs.eve_image }}
```

Comment:
As I remember, we expect only something like `11.0.5-lts` as `EVE_VERSION` variable in [test](https://github.com/lf-edge/eden/blob/4fbc73cb9e93fc51416b54662b739ffc2f98f98a/tests/update_eve_image/testdata/update_eve_image_http.txt#L2). At the same time registry may be not aligned with define in test. So it looks like additional adjustment in test required.